extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**group**|`string`|The group this content belongs to. The value must be a group ID declared in the `groups` dictionary.|No|
+|**extensions**|`any`||No|
+|**extras**|`any`||No|
+
+Additional properties are allowed.
+
+<!-- omit in toc -->
+#### Content extension.group
+
+The group this content belongs to. The value must be a group ID declared in the `groups` dictionary.
+
+* **Type**: `string`
+* **Required**: No
+
+<!-- omit in toc -->
+#### Content extension.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+<!-- omit in toc -->
+#### Content extension.extras
+
+* **Type**: `any`
+* **Required**: No
+
+<!-- omit in toc -->
+## `3DTILES_implicit_tiling` subtree extension
+* [`3DTILES_implicit_tiling` subtree extension](#reference-3dtiles_metadata-extension-for-3dtiles_implicit_tiling-subtree) (root object)
+* [`Subtree property`](#reference-subtree-property)
+
+
+---------------------------------------
+<a name="reference-3dtiles_metadata-extension-for-3dtiles_implicit_tiling-subtree"></a>
+<!-- omit in toc -->
+### `3DTILES_implicit_tiling` subtree extension
+
+An object assigning tile metadata to implicit tiles in the subtree. Tile metadata only exists for available tiles and is tightly packed by increasing tile index. To access individual tile metadata, implementations may create a mapping from tile indices to tile metadata indices.
+
+**`3DTILES_implicit_tiling subtree extension` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**class**|`string`|The class that tile metadata conforms to. The value must be a class ID declared in the `classes` dictionary.|No|
+|**properties**|`object`|A dictionary, where each key corresponds to a property ID in the class' `properties` dictionary and each value is an object describing where property values are stored. Optional properties may be excluded from this dictionary.|No|
+|**extensions**|`any`||No|
+|**extras**|`any`||No|
+
+Additional properties are allowed.
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.class
+
+The class that tile metadata conforms to. The value must be a class ID declared in the `classes` dictionary.
+
+* **Type**: `string`
+* **Required**: No
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.properties
+
+A dictionary, where each key corresponds to a property ID in the class' `properties` dictionary and each value is an object describing where property values are stored. Optional properties may be excluded from this dictionary.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: `subtree.property`
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.extras
+
+* **Type**: `any`
+* **Required**: No
+
+
+
+
+---------------------------------------
+<a name="reference-subtree-property"></a>
+<!-- omit in toc -->
+### Subtree property
```

Comment:
I'm a bit confused; when I read the **Concepts** section, I assumed that properties could be backed by inline JSON or buffers - now it looks like buffers are just for subtrees.  Whatever the design is is probably fine, but the delivery can be more explicit.